src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -199,10 +208,13 @@
 .color-editor section .slider.opacity-slider {
     background-image: url("../img/color_thumb_back.png");
     background-size: 8px;
-    margin-right: 0px;  
+    margin-right: 0px;
```

Comment:
Revert this change, please.  Spurious whitespace-only change are to be avoided, since they make blame info harder to untangle.